### PR TITLE
Documentation Cleanup 4

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed
Formatting of documentation.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by oscargus** [comment]
This seems to be passing now.

File: sympy/core/expr.py [hunk 1]
```diff
@@ -3953,7 +3953,7 @@ def expr_free_symbols(self):
 
 
 def _mag(x):
-    """Return integer ``i`` such that .1 <= x/10**i < 1
+    r"""Return integer $i$ such that $0.11 \le x/10^i < 1$
```
Comment (oscarbenjamin) [inline sympy/core/expr.py:3956]: Shoudl this be `0.11` or `0.1`?

File: sympy/functions/elementary/complexes.py [hunk 2]
```diff
@@ -1073,10 +1073,10 @@ def _eval_Abs(self):
 
 
 class periodic_argument(Function):
-    """
+    r"""
     Represent the argument on a quotient of the Riemann surface of the
     logarithm. That is, given a period $P$, always return a value in
-    (-P/2, P/2], by using exp(P*I) == 1.
+    $(-P/2, P/2]$, by using $\exp(PI) = 1$.
```
Comment (oscarbenjamin) [inline sympy/functions/elementary/complexes.py:1079]: A space between P and I might help when reading this in the code/help.

File: sympy/core/expr.py (hunk 1 above)
Comment (oscarbenjamin) [reply sympy/core/expr.py:3956]:
```suggestion
    r"""Return integer $i$ such that $0.1 \le x/10^i < 1$
```

**Comment by oscarbenjamin** [comment]
Looks good!

**Pull request merged**